docs/modules/operation/pages/events/event-tokens.adoc

```diff
@@ -0,0 +1,118 @@
+%descr%::
+    The event description.
+%logmsg%::
+    The event logmsg.
+%time%::
+    The time of the event.
+%shorttime%::
+    The time of the event formatted using DateFormat.SHORT for a completely numeric date/time.
+%dpname%::
+    Formerly used for the ID of the Minion that the event was received on.
+    This placeholder is deprecated and will be resolved to an empty string.
+%nodeid%::
+    The numeric node ID of the device that caused the event, if any.
+%nodelabel%::
+    The node label for the node given in `%nodeid%`, if available.
+%nodelocation%::
+    The node location for the node given in `%nodeid%`, if available.
+%host%::
+    The host at which the event was generated.
+%interface%::
+    The IP interface associated with the event, if any.
+%foreignsource%::
+    The requisition name for the node given in `%nodeid` if available.
+%foreignid%::
+    The requisition ID for the node given in `%nodeid` if available.
+%ifindex%::
+    The interface's SNMP ifIndex.
+%interfaceresolve%::
+    Does a reverse lookup on the `%interface%` and returns its name, if available.
+%service%::
+    The service associated with the event, if any.
+%severity%::
+    The severity of the event.
+%snmphost%::
+    The host of the SNMP agent that generated the event.
+%id%::
+    The SNMP enterprise OID for the event.
+%trapoid%::
+    Full trap OID for the event.
+%idtext%::
+    The decoded (human-readable) SNMP Enterprise OID for the event.
+%ifalias%::
+    The interface's SNMP ifAlias.
+%generic%::
+    The generic trap-type number for the event.
+%specific%::
+    The specific trap-type number for the event.
+%community%::
+    The community string for the trap.
+%version%::
+    The SNMP version of the trap.
+%snmp%::
+    The SNMP information associated with the event.
+%operinstruct%::
+    The operator instructions for the event.
+%mouseovertext%::
+    The mouse over text for the event.
+%tticketid%::
+    The trouble ticket ID associated with the event, if available.
+%primaryinterface%::
+The primary interface IP address for the node given in `%nodeid%`, if available.
+
+CAUTION: The use of multiple Minions in one location can break the alarm life-cycle for a some {page-component-title} features.
+To avoid this problem, the `%dpname%` value can always be replaced by an empty string by setting `org.opennms.netmgt.eventd.cleardpname=true` in `$\{OPENNMS_HOME}/etc/opennms.properties.d/events.properties`.
+
+== Asset tokens
+A node may have additional asset records stored for it.
+You can access these records using the `asset` replacement token, which takes the form:
+
+%asset[<token>]%::
+    The value of the asset field named `token`, or "Unknown" if it does not exist.
+
+== Hardware tokens
+A node may have additional hardware details stored for it.
+You can access these details using the `hardware` replacement token, which takes the form:
+
+%hardware[<token>]%::
+    The hardware field `token`'s value.
```

Comment:
"token" does not render properly in the built version. We just see the first accent, and no formatting. Probably conflicting with the apostrophe after.